src/effects.js

```diff
@@ -27,19 +27,29 @@ var
 	rfxtypes = /^(?:toggle|show|hide)$/,
 	rrun = /queueHooks$/;
 
-function raf() {
+function raf( timestamp ) {
 	if ( timerId ) {
 		window.requestAnimationFrame( raf );
-		jQuery.fx.tick();
+		jQuery.fx.tick( timestamp );
 	}
 }
 
+// We need to be using jQuery.now() or performance.now() consistently as they return different
+// values: performance.now() counter starts on page load.
+// Support: IE <10, Safari <8.0, iOS <9, Android <4.4, Node with jsdom 9.4
+function getTimestamp() {
```

Comment:
Given the comments, shouldn't we have something more like this?
```js
getTimestamp = window.performance && typeof window.performance.now === "function" ?

	// or window.performance.now.bind( window.performance )
	jQuery.proxy( window.performance, "now" ) :

	jQuery.now;
```
If the complaint is about testing, then we just need an iframe test in which the sandbox is mocked before loading jQuery.